service-wfs3/src/test/java/org/oskari/service/wfs3/WFS3ServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package org.oskari.service.wfs3;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.oskari.service.wfs3.model.WFS3CollectionInfo;
+import org.oskari.service.wfs3.model.WFS3Exception;
+
+public class WFS3ServiceTest {
+
+    @Test
+    @Ignore("Depends on outside service")
+    public void testJSONWorks() throws WFS3Exception, IOException {
+        String url = "https://beta-paikkatieto.maanmittauslaitos.fi/geographic-names/wfs3/v1/";
+        WFS3Service expected = WFS3Service.fromURL(url);
```

Comment:
Can we add a response as test resource so we can test parsing and "document" an expected response from server?